transport/grpc/options.go

```diff
@@ -90,6 +92,15 @@ func Logger(logger *zap.Logger) TransportOption {
 	}
 }
 
+// Meter sets a meter to use for transport metrics.
+//
+// The default is to not emit any metrics.
```

Comment:
This looks to be a setter option method, is this the right place for this comment?

Reply:
Added a comment inline with Logger setter:
```
// Logger sets a logger to use for internal logging.
//
// The default is to not write any logs.
```